ui/DesignSystem/Primitive/Avatar.svelte

```diff
@@ -46,15 +84,47 @@
     height: 64px;
   }
 
-  div {
+  .container {
     display: flex;
     align-items: center;
     justify-content: center;
   }
+
+  .avatar {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
 </style>
 
-<div class={size} {style}>
-  <img class={avatarClass} src={imageUrl} alt="user-avatar" />
+<div class={`container ${size}`} {style}>
+  {#if imageUrl}
```

Comment:
Another case where we would want to render the fallback is if the image can't be loaded. We could do that in a follow-up where we always render the fallback behind the actual image container.

Reply:
Good idea! I added `display: none` on error. This means though that one has to pass the fallback avatar as well.

Reply:
That sounds like a worthwhile requirement anyway, to always pass the fallback.